### ISSUE TITLE
AdguardHome DNS Graphs Blank

### ISSUE THREAD

**Issue by samcro1967** [glanceapp/glance#211]
The DNS widget seems to be connected to both my AdguardHome servers and is retuning some data, but the graphs are blank.  It has been running for several hours now.  Not sure if it just needs to run longer, I am missing something, or I have done something wrong.

**glance DNS Widgets for 2 AdguardHome servers**
![image](https://github.com/user-attachments/assets/6de6c15b-b652-4240-8016-535a3a752a3b)

**AdguardHome Stastics from 1 of the servers**
![image](https://github.com/user-attachments/assets/5454522e-bcda-42bf-8a40-0c0a2a902754)

**Relevant glance.yml config**
```
  - name: Monitoring
    columns:
      - size: full
        widgets:
          - type: dns-stats
            title: server1
            service: adguard
            url: http://192.168.1.106:8126/
            username: <redacted>
            password: <redacted>
      - size: full
        widgets:
          - type: dns-stats
            service: adguard
            title: server2
            url: http://192.168.1.108:8126/
            username: <redacted>
            password: <redacted>
```

**Comment by svilenmarkov** [comment]
Thanks for reporting this! 

I think I know what the issue is. I naively assumed that AdGuard always returns data for the last 24 hours since that's what Pi-hole does, but I had just coincidentally set my statistics retention to 24 hours. Could you please confirm that your statistics retention is set to more than 24 hours if you go to `Settings -> Statistics configuration`?

**Comment by samcro1967** [comment]
It is set to 90 days:
![image](https://github.com/user-attachments/assets/301d489a-4b3e-4540-b7db-e9a48dd50956)

**Comment by svilenmarkov** [comment]
I've just released `v0.6.1` which should fix this.

**Comment by samcro1967** [comment]
Can confirm it is now working.
![image](https://github.com/user-attachments/assets/c924fdeb-ef06-48fd-9108-81da65852c9d)
